jenkins/fossa_run.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+run_fossa()
+{
+  echo "Queuing FOSSA Checks..."
+  if ./fossa; then
+    echo "FOSSA Checks Queued Successfully!"  
+    fail_build_check
+  else
+    echo "FOSSA Checks Failed to Queue :("
+    exit 1
+  fi
+}
+
+fail_build_check()
+{
+  echo "Determining FOSSA Fail Build Flag..."
+  
+  # Check for Fail Build Toggle, default to false
+  export FOSSA_FAIL_BUILD=${FOSSA_FAIL_BUILD:-false}
+  # Get Toggle value in lowercase
+  export FOSSA_FAIL_BUILD_TOGGLE=$(echo "$FOSSA_FAIL_BUILD" | tr '[:upper:]' '[:lower:]')
+
+  # Check if Fail Build Toggle is Enabled
+  if [ "$FOSSA_FAIL_BUILD_TOGGLE" == "true" ]; then
+    # Return FOSSA CLI Exit Code for Build Failures
+    echo "FOSSA Fail Build Flag Enabled.  Checking for Policy Violations..."
+    export FOSSA_EXIT_CODE=$(./fossa test)
+    echo "FOSSA Policy Violations check returned : $FOSSA_EXIT_CODE (non-zero codes are failures)"
+    exit $FOSSA_EXIT_CODE
+  else
+    echo "FOSSA Fail Build Flag Disabled.  Skipping Policy Violations Check!"
+  fi
+}
+
+# Check for Build Node Index
+if [ -z $CI_NODE_INDEX ]; then
```

Comment:
At present we have only a single worker node across CTMS (PR Build node) - this should be ok, as only one commit in queue will be under execution

Reply:
Agree , we have only single threaded build but good to see these scripts will support multi thread build for future extension.